ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -816,17 +816,19 @@ protocolInfoCardano protocolParamsByron@ProtocolParamsByron {
 
         register ::
              (EraCrypto era ~ c, ShelleyBasedEra era)
-          => Flip LedgerState mk (ShelleyBlock proto era)
-          -> Flip LedgerState mk (ShelleyBlock proto era)
-        register (Flip st) = Flip st {
+          => Flip LedgerState ValuesMK (ShelleyBlock proto era)
+          -> Flip LedgerState ValuesMK (ShelleyBlock proto era)
+        register (Flip st) = Flip $ unstowLedgerTables $ forgetLedgerTables $ st {
```

Comment:
I think what's happening here is not trivial, so I wonder if we should comment why do we do the `unstow;stow` sequence.

Reply:
Added a comment